paypalscript.py

```diff
@@ -1,23 +1,16 @@
+#!/usr/bin/env python3
 import csv
-ldate = ["Date"]
-lname = ["Name"]
-lammo = ["Ammount"]
-ldesc = ["Description"]
-with open("Paypal Export.CSV", 'r', encoding="utf-8-sig") as filein:
-    csv_filein = csv.DictReader(filein)
-    for rowi in csv_filein:
-        if rowi["Balance Impact"] == "Credit":
-            ldate.append(rowi["Date"])
-            lname.append(rowi["Name"])
-            lammo.append(rowi["Net"])
-            ldesc.append(rowi["Type"])
-    filein.close()
 
+mapping = {"Date": "Date", "Name": "Name", "Amount": "Net", "Description": "Type"}
+
+def strip_row(row):
+    return {k: row[v] for k,v in mapping.items()}
 
-with open('output.csv', 'w', newline='') as fileout:
-    writer = csv.writer(fileout)
-    rcount = 0
-    for row in ldate:
-        writer.writerow((ldate[rcount], lname[rcount], lammo[rcount], ldesc[rcount]))
-        rcount = rcount + 1
-    fileout.close()
+with open("Paypal Export.csv", 'r') as filein:
```

Comment:
This got dropped from the original and is needed to deal with paypal's export bullshit:  encoding="utf-8-sig"

Reply:
Whoops, I was working off of the original Bradley sent me. Second patch adds it back.